modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -34,9 +34,11 @@ def claim_attributes
       def service_information
         info = @data[:serviceInformation]
         service_periods = format_service_periods(info&.dig(:servicePeriods))
+        confinements = format_confinements(info&.dig(:confinements))
```

Comment:
@mchristiansonVA  I think we need a check here to make sure there are confinements present first, otherwise we are calling `.each` on nil.

Reply:
Good catch! I'll add a check...